p256/parameters.ml

```diff
@@ -7,3 +7,6 @@ let b =
 let g =
   `Hex
     "046B17D1F2E12C4247F8BCE6E563A440F277037D812DEB33A0F4A13945D898C2964FE342E2FE1A7F9B8EE7EB4A7C0F9E162BCE33576B315ECECBB6406837BF51F5"
+
+let p =
+  `Hex "ffffffff00000001000000000000000000000000ffffffffffffffffffffffff"
```

Comment:
Super minor note here : other parameters are uppercase, we can probably have this one uppercase too.

Reply:
let's do this!